tasks/libs/pipeline/data.py

```diff
@@ -1,18 +1,33 @@
 import re
 from collections import defaultdict
 
-from gitlab.v4.objects import ProjectJob
+from gitlab.v4.objects import ProjectCommit, ProjectJob, ProjectPipeline
 
 from tasks.libs.ciproviders.gitlab_api import get_gitlab_repo
 from tasks.libs.types.types import FailedJobReason, FailedJobs, FailedJobType
 
 
-def get_failed_jobs(project_name: str, pipeline_id: str) -> FailedJobs:
+def get_commit(project_name: str, git_ref: str) -> ProjectCommit:
     """
-    Retrieves the list of failed jobs for a given pipeline id in a given project.
+    Retrieves the git ref for a given pipeline id in a given project.
+    """
+    repo = get_gitlab_repo(project_name)
+    return repo.commits.get(git_ref)
+
+
+def get_pipeline(project_name: str, pipeline_id: str) -> ProjectPipeline:
+    """
+    Retrieves the git ref for a given pipeline id in a given project.
     """
     repo = get_gitlab_repo(project_name)
-    pipeline = repo.pipelines.get(pipeline_id)
+    return repo.pipelines.get(pipeline_id)
```

Comment:
Should this be within `gitlab_api.py` ?

Reply:
Sounds good, let me move these two methods.